### ISSUE TITLE
Error after updating HA to 0.106

### ISSUE THREAD

**Issue by cacelotto** [gurbyz/power-wheel-card#35]
I have read the wiki about what to try first before creating a bug report.

**Which custom card does give you issues?**
power-wheel-card

**Which version of HA do you use?** Home Assistant 0.106.0

**Which version of the card do you use?**  const __VERSION = "0.0.16";

**Describe the bug**
A clear and concise description of what the bug is. Paste any error messages.
Like the bug opened by VDRainer. Initially I had v.0.0.15 installed. I understood that installing version 0.0.16 all would have worked nicely.

**Configuration of the card**
Copy the complete configuration of the card, even if you think it isn't relevant.
```
- type: custom:power-wheel-card
      solar_power_entity: sensor.inverter_current_power
      grid_power_consumption_entity: sensor.inverter_current_export_power
      grid_power_production_entity: sensor.inverter_current_import_power 
```

**Dev console debug output**
Add `debug: true` to your card parameters. Copy the complete output of the card that is logged to your dev console.
Sorry, I don't know what the dev console is.

**Expected behavior**
All worked well since end of november, when my FV panels have been activated. This card is fantastic.

**Screenshots**
![power-wheel-Home Assistant](https://user-images.githubusercontent.com/57525590/75445274-663c0580-5965-11ea-8892-460d35c70412.png)

**Additional context**
no additional info

**Comment by Ysbrand** [comment]
Same here fails to work after updating to 0.106.0

- type: custom:power-wheel-card
    title: Stroom voorziening
    solar_power_entity: sensor.toon_p1_power_solar
    # grid_power_consumption_entity: sensor.toon_p1_power_use_high
    grid_power_consumption_entity: sensor.grid_power_use_total
    # grid_power_production_entity: sensor.toon_p1_power_prod_high
    grid_power_production_entity: sensor.grid_power_prod_total
    update_interval: 60   

Error:
Cannot add property production_is_positive, object is not extensible
type: 'custom:power-wheel-card'
title: Stroom voorziening
solar_power_entity: sensor.toon_p1_power_solar
grid_power_consumption_entity: sensor.grid_power_use_total
grid_power_production_entity: sensor.grid_power_prod_total
update_interval: 60

**Comment by gurbyz** [comment]
Hi, I suspect something is tricking you with serving an old version of the card. Could be the old version that's still in the cache or could be a HACS issue that I'm not aware of. There is a user reporting a workaround for the HACS issue here: https://community.home-assistant.io/t/lovelace-power-wheel-card/82374/310?u=gurbyz Wyx087 says: "I pressed Reinstall in HACS and the card started to work after a refresh."

Could you try that (if you were using HACS)?

**Comment by Ysbrand** [comment]
Confirmed, reinstall fixes the problem. Thanks for that!

**Comment by cacelotto** [comment]
Maybe a combination of coincidences: reinstalled HACS and PWC: nothing happened. Shut down and restarted manually my raspberry: nothing happened. Then I noticed the PWC plugin named url: **/community_plugin**/power-wheel-card/power-wheel-card.js; instead I found the file both in **www** and community directories. Also **ui-lovelace** used the www url instead of the community one. Probably I installed the plugin manually and forgot to change the url when activated it through HACS.

Now it works. Thank you for the time you wasted running behind my mistakes.
For me the issue is closed now.
